lib/consumers.js

```diff
@@ -180,18 +180,17 @@ ConsumerStream.prototype.read = function() {
                 for(var i = 0; i < msgs.length; i++) {
                     decoded_msgs.push(this.schemaClass.decodeMessage(msgs[i]));
                 }
-                this.emit('read', decoded_msgs);
```

Comment:
I think the incompatibility is fine -- there's a reason I kept the version number small and this is strictly an improvement. I also mentioned we might want to just implement the `Readable` interface here: http://www.confluent.io/blog/getting-started-with-kafka-in-node.js-with-the-confluent-rest-proxy